rest/aggregates/aggregates.go

```diff
@@ -0,0 +1,45 @@
+package aggregates
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/polygon-io/client-golang/rest/client"
+)
+
+// todo: add comments for godoc
+
+const (
+	GetPath               = "/v2/aggs/ticker/{ticker}/range/{multiplier}/{resolution}/{from}/{to}"
+	GetPreviousClosePath  = "/v2/aggs/ticker/{ticker}/prev"
+	GetGroupedDailyPath   = "/v2/aggs/grouped/locale/{locale}/market/{marketType}/{date}"
+	GetDailyOpenClosePath = "/v1/open-close/{ticker}/{date}"
+)
+
+type Client struct {
+	client.HTTPBase
+}
+
+func (ac *Client) Get(ctx context.Context, params GetParams, opts ...client.Option) (*AggsResponse, error) {
```

Comment:
We're leaving out the resource type in method names since users will call these methods with the package name as a prefix (e.g. `aggregates.Get(...)`). Open to changing it back to our current pattern though (`GetAggs`).